.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,98 @@
+name: Continuous integration
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint_and_typecheck:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r build/test-requirements.txt
+    - name: Lint with flake8
+      run: |
+        flake8 .
+    - name: Type check with mypy
+      run: |
+        mypy jax
+
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - python-version: 3.6
+            os: macos-latest
+            enable-x64: disable-x64
+            num_generated_cases: 10
```

Comment:
It's hard to tell, but looks like this job is timing out after an hour. Hopefully it's not hanging... Regardless, I would be fine checking in everything but this job for now.

Reply:
I'm wondering if this was a one-off flake because I'm also monkeying with the configuration. Let's see what happens with the new push I just made.

Reply:
It seemed to get stuck again, so I disabled the mac build.